README.md

```diff
@@ -36,6 +36,7 @@ Run:
 ```
 git clone https://github.com/CoderDojo/cp-local-development.git && cd cp-local-development
 ./setup_repo.sh
+yarn
```

Comment:
why not just the post-install script at the end of the setup_repo.sh

Reply:
Fair enough on not using yarn, updated as such.

I wanted a way of just quickly installing/updating node_modules across all repos. I've made it a separate script instead of using a postinstall though.